pulpcore/pulpcore/constants.py

```diff
@@ -36,4 +37,22 @@
     (SYNC_MODES.MIRROR, 'Add new content and remove content is no longer in the remote repository.')
 )
 
+
+POLICY_MODE = SimpleNamespace(
+    IMMEDIATE='immediate',
+    ON_DEMAND='on_demand',
+    CACHE_ONLY='cache_only',
+)
+
+POLICY_CHOICES = (
```

Comment:
Same here.

Suggest: `DOWNLOAD_POLICIES` or `DOWNLOAD_POLICY_CHOICES`

Reply:
fixed in next push.